src/components/UserProfile/UserFavorites.tsx

```diff
@@ -0,0 +1,16 @@
+import { faHeart } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+function UserFavorites() {
+  return (
```

Comment:
We can put this inside `favorites/page.tsx`. No need for a new component now, that is read only.